### PR TITLE
[INFINITY-1961] Very simple script. Simply extends the lifetime of a CCM cluster assuming that cluster has a cluster_info.json in the same directory.

### PR DESCRIPTION
**Pull request by benclarkwood**

### CODE REVIEW

**Comment by benclarkwood** [comment]
Lol. The PR jobs were overridden in such a way that they weren't going to run the extension script. Fixing that.

File: tools/ccm_extend.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+# Get all the clusters
+headers = {'Authorization': 'Token {}'.format(ccm_token)}
+
+all_clusters = json.loads(requests.get(base_url+'/api/cluster/active/all', headers=headers).text)
+for cluster in all_clusters:
+    if cluster['name'] == cluster_name:
+        cluster_id = cluster['id']
+
+response = requests.put(base_url+'/api/cluster/{}/'.format(cluster_id), data = {'time':'240'}, headers=headers)
```
Comment (gabrielhartmann) [inline tools/ccm_extend.py]: Comment what this API call does please.

**Comment by gabrielhartmann** [comment]
@benclarkwood:  Why did `helloworld` timeout again?

**Comment by benclarkwood** [comment]
There was also a fun 130 minute limit on the HW PR build. Fixed that.

**Pull request merged**